testing/chrome-gnome-shell/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=chrome-gnome-shell
+pkgver=10.1
+pkgrel=0
+pkgdesc="GNOME Shell integration for WebExtensions capable browsers"
+url="https://wiki.gnome.org/Projects/GnomeShellIntegrationForChrome"
+arch="noarch"
+license="GPL-3.0-or-later"
+depends="py3-requests py3-gobject3"
+makedepends="cmake gettext-dev jq python3"
+options="!check" # No tests available
+source="https://download.gnome.org/sources/chrome-gnome-shell/${pkgver}/chrome-gnome-shell-${pkgver}.tar.xz"
+
+build() {
+	if [ "$CBUILD" != "$CHOST" ]; then
+		CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
+	fi
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		-DBUILD_EXTENSION=FALSE \
+		${CMAKE_CROSSOPTS}
+	make
+}
+
+check() {
```

Comment:
```suggestion

```

If there is no test suite then please remove the check().

Reply:
Hm, but why? Once they've added tests we can just flick the switch (`options="!check"`) and it'll work.

Reply:
Will have to ask higher authority on why...

I have had PRs before with the same reasoning to keep it, but I was asked to remove it.

One reason could be that keeping could cause confusion.

Reply:
Ah, alright. I mean I don't mind removing it, but I like to know the reason for my doing :)